### DISCUSSION TITLE
Ollama missing in Helper-Scripts.com

### DISCUSSION THREAD

**Discussion by chpego** [community-scripts/ProxmoxVE#201] [Q&A] [answered]
it's available to start / install from :
``
bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/ollama.sh)"
``
but missing in the website : 
![image](https://github.com/user-attachments/assets/f8ecd260-f486-4904-b056-afc5fc4e3371)

**Comment by newzealandpaul** [comment, marked as answer]
Thank you for reporting. This has been converted into an issue:

https://github.com/community-scripts/ProxmoxVE/issues/202
